src/sqlfluff/rules/L052.py

```diff
@@ -160,23 +185,29 @@ def _eval(self, context: RuleContext) -> Optional[LintResult]:
                     # If preceding segments are found then delete the old
                     # semi-colon and its preceding whitespace and then insert
                     # the semi-colon in the correct location.
-                    fixes = [
-                        LintFix.replace(
-                            anchor_segment,
-                            [
-                                anchor_segment,
-                                SymbolSegment(raw=";", type="symbol", name="semicolon"),
-                            ],
-                        ),
-                        LintFix.delete(
-                            context.segment,
-                        ),
-                    ]
-                    fixes.extend(LintFix.delete(d) for d in whitespace_deletions)
-                    return LintResult(
-                        anchor=anchor_segment,
-                        fixes=fixes,
-                    )
+                    self.logger.debug("case 1")
```

Comment:
Presume you're happy to leave these debug statements in?

Reply:
I am. They are similar in principle to some of the logging in L003. Less descriptive, but helpful for finding the problematic code in case of future issues.